lib/hint.rb

```diff
@@ -44,7 +44,10 @@ def which_machinery
     end
 
     def get_started(_options)
-      "You can get started by inspecting a system. Run:\n#{program_name} inspect HOSTNAME"
+      "You can get started by inspecting a system. Run:\n'#{program_name} inspect HOSTNAME'\n" \
+      "Specify user for connecting via SSH:\n" \
+      "'#{program_name} inspect --remote-user USER HOSTNAME'\n" \
+      "(Note: specified user needs sudo access or root on remote machine)"
```

Comment:
What do you mean by `or root on remote machine` ?

Sudo is mandatory for each user except of root but if a user wants to inspect as root they do not need the option so I would not mention it here.